tree/ntuple/v7/inc/ROOT/RMiniFile.hxx

```diff
@@ -107,7 +114,10 @@ public:
 
 private:
    struct RFileProper {
+      /// If fDirectory is set, fFile is equal to fDirectory->GetFile()
       TFile *fFile = nullptr;
```

Comment:
Why do we need this? Can't we use `fDirectory` in either case?

Reply:
`TFile` has some file specific methods that we need, such as `Seek()` and `WriteBuffer()`.

Reply:
Ah, yes that makes sense.